chall01/vgrankul.py

```diff
@@ -0,0 +1,50 @@
+#!/bin/python3
+import sys
+
+sos = [
+".-",
+"-...", 
+"-.-.", 
+"-..", 
+".", 
+"..-.", 
+"--.", 
+"....", 
+"..", 
+".---", 
+"-.-", 
+".-..", 
+"--", 
+"-.", 
+"---", 
+".--.", 
+"--.-", 
+".-.", 
+"...", 
+"-", 
+"..-", 
+"...-", 
+".--", 
+"-..-", 
+"-.--", 
+"--.."]
+
+i = 0
+letter = ['a', 'b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
+Bletter = ['A', 'B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']
+sos_string = ""
+
+if len(sys.argv) != 2:
+    print("usage: ./vgrankul.py <a-zA-Z string>")
+else:
+    string = sys.argv[1]
+    if string.replace(' ', '').isalpha() == False:
+        print("usage: ./vgrankul.py <a-zA-Z string>")
+        exit()
```

Comment:
It's the perfect way to solve this.

However, you could have written:

```python
if not string.replace(' ', '').isalpha()
```
Plus, the convention about printing `usage` is:
- if you have to write the usage because of an input error/user error you print the usage to stderr
- if the user as request help with for example : /yourScript.py --help then you print it to stdout.